ci/module-kmm-ci-build.template.yaml

```
@@ -10,7 +10,7 @@ spec:
         moduleName: kmm_ci_a
       kernelMappings:
         - literal: KVER_CHANGEME
-          containerImage: registry.minikube/kmm-kmod:local
+          containerImage: registry.ci.openshift.org/rh-ecosystem-edge/kmm-kmod:local
```

I think we rather want to push the image to the internal OpenShift Registry?
```suggestion
          containerImage: image-registry.openshift-image-registry.svc:5000/default/kmm-kmod:local
```